timestampvm/src/vm/mod.rs

```diff
@@ -291,11 +314,14 @@ impl snow::engine::common::vm::Vm for Vm {
 }
 
 #[tonic::async_trait]
-impl subnet::rpc::snowman::block::ChainVm for Vm {
+impl<A> snowman::block::ChainVm for Vm<A>
+where
+    A: AppSender + Send + Sync + Clone + 'static,
+{
+    type Block = Block;
+
     /// Builds a block from mempool data.
-    async fn build_block(
-        &self,
-    ) -> io::Result<Box<dyn subnet::rpc::consensus::snowman::Block + Send + Sync>> {
+    async fn build_block(&self) -> io::Result<<Self as snowman::block::ChainVm>::Block> {
```

Comment:
Any reason we don't import `ChainVm` here? Keeping types behind multiple modules really hurts readability.

Reply:
yeah I guess we have done that just so folks can understand the namespace but I agree its probably not all that helpful

Reply:
cleaned this all up